src/utils/components/ThirdPartyDisclaimer.vue

```diff
@@ -50,7 +54,9 @@ export default {
             currentLang: (state) => state.i18n.lang,
         }),
         tooltipContent() {
-            return this.$t('external_data_tooltip')
+            return this.isExternalDataLocal
```

Comment:
I don't think a `this.isExternalDataLocal` still exists, as you've renamed the prop to `isLocalFile`